SyncUps-StackBased/SyncUps/App.swift

```diff
@@ -18,6 +19,8 @@ class AppModel {
   @ObservationIgnored
   @Dependency(\.uuid) var uuid
 
+  @CasePathable
+  @dynamicMemberLookup
   enum Destination: Hashable {
```

Comment:
Interestingly, we don't leverage these attributes in the app code, but it helps with testing.